src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1770,66 +1770,89 @@ public static void ConsumeLotsOfComments(string valueString, bool insideArray, s
             Assert.Equal(dataUtf8.Length, json.BytesConsumed);
         }
 
+        struct SingleCommentVerifyInfo
```

Comment:
This seems unnecessary, especially for the start/end object.

If we want to keep it, I would add private access modifier and move it to the end of the file rather than in the middle of the tests.